### PR TITLE
add list of projects to the title of the search page

### LINKED ISSUE #1365: add list of searched projects to the title of search results page
As noted in `PageConfig#getSearchTitle()`, the list of projects can be added to the title. Perhaps good idea would be to trim them (and append `...`) if the list is too long (say 10 projects).

### PR DESCRIPTION
**Pull request by vladak**
fixes #1365

### CODE REVIEW

File: src/org/opensolaris/opengrok/web/PageConfig.java [hunk 1]
```diff
@@ -1340,6 +1340,14 @@ public String getShortPath(String path) {
         return file.getName();
     }
 
+    private String addTitleDelimiter(String title) {
```
Comment (tulinkry) [inline src/org/opensolaris/opengrok/web/PageConfig.java]: copyright

**Comment by tulinkry** [comment]
Looks good

File: src/org/opensolaris/opengrok/web/PageConfig.java (hunk 1 above)
Comment (vladak) [inline src/org/opensolaris/opengrok/web/PageConfig.java]: fixed

File: src/org/opensolaris/opengrok/web/PageConfig.java [hunk 2]
```diff
@@ -1353,31 +1361,35 @@ public String getSearchTitle() {
...
         }
 
-        // TODO: possibly add projects too
+        if (req.getParameterValues(QueryBuilder.PROJECT) != null && req.getParameterValues(QueryBuilder.PROJECT).length != 0) {
+            if (!title.isEmpty()) {
+                title += " ";
+            }
+            title += "in projects: ";
+            String projects[] = req.getParameterValues(QueryBuilder.PROJECT);
+            for (int i = 0; i < projects.length; i++) {
```
Comment (tulinkry) [inline src/org/opensolaris/opengrok/web/PageConfig.java]: Sorry that I didn't see this during the first review. In Java8 I think there is a possibility to make this shorter. Something like:
```java
Arrays.asList(a).stream().collect(Collectors.joining(","))
```

Comment (vladak) [inline src/org/opensolaris/opengrok/web/PageConfig.java]: good suggestion, thanks !

Comment (tulinkry) [inline src/org/opensolaris/opengrok/web/PageConfig.java]: I'm not sure but I suspect that raw "," is without the space between the elements

File: src/org/opensolaris/opengrok/web/PageConfig.java [hunk 3]
```diff
@@ -1353,31 +1362,30 @@ public String getSearchTitle() {
...
         }
 
-        // TODO: possibly add projects too
+        if (req.getParameterValues(QueryBuilder.PROJECT) != null && req.getParameterValues(QueryBuilder.PROJECT).length != 0) {
+            if (!title.isEmpty()) {
+                title += " ";
+            }
+            title += "in projects: ";
+            String projects[] = req.getParameterValues(QueryBuilder.PROJECT);
+            title += Arrays.asList(projects).stream().collect(Collectors.joining(","));
```
Comment (tulinkry) [inline src/org/opensolaris/opengrok/web/PageConfig.java]: I'm not sure but I suspect that the raw "," is without the space between the elements

Comment (vladak) [inline src/org/opensolaris/opengrok/web/PageConfig.java]: yep, I realized that too and decided it looks better that way.

**Pull request merged**